app/lambda/create_session/lambda_function.py

```diff
@@ -0,0 +1,65 @@
+import json
+import sys
+import time
+import uuid
+
+sys.path.append("../")
+import boto3
+from lambda_base import LambdaBase
+
+
+class CreateSessionLambda(LambdaBase):
+    def __init__(self): 
+        pass
+
+    def CurTimeToEpoch(self): 
+        return int(time.time())
+
+    def EpochToCurTime(self, epoch): #int
+        return time.strftime('%Y-%m-%d %H:%M:%S', time.localtime(epoch))
+
+    def putItem(self, PatientId, HealthCareProfessionalId, SessionName, TimeStampStart):
+        client = boto3.resource('dynamodb')
+        table = client.Table("Sessions")
+        tempId = uuid.uuid4().hex 
+        epochTime = self.CurTimeToEpoch()
+        SessionId = "s-"+tempId+str(epochTime)
+        ComprehendS3Path = "c-"+tempId+str(epochTime)
+        TranscribeS3Path = "t-"+tempId+str(epochTime)
+        AudioS3Path = "a-"+tempId+str(epochTime)
+        table.put_item(Item= {'SessionId': SessionId,
```

Comment:
Please consider creating a constants file where you can store all the string values that are referred across the solution (like column names - SessionId, PatientID etc.)
This makes development less prone to typo errors in the code. It also makes it easy to refactor these values when needed


For example :  

constants.py

DATASTORE_COLUMN_PATIENT_ID = 'PatientId'
DATASTORE_COLUMN_SESSION_ID = 'SessionId'

Reply:
Done.